owncloudApp/src/main/res/layout/uploader_layout.xml

```diff
@@ -32,59 +32,53 @@
 
         <include layout="@layout/owncloud_toolbar" />
 
-        <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Why removing this wrapping layout? Not bad to remove it if it keeps code simpler and with the same functionality, just curiosity 👍

Reply:
Yes, I removed the constraint layout because it maintained the same functionality and was easier to order. In addition, in this way we skip a component.